.vscode/settings.json

```diff
@@ -15,8 +15,7 @@
     "Dockerfile.*": "dockerfile",
     ".releaserc": "json"
   },
-  "omnisharp.autoStart": false,
```

Comment:
please don't remove this

Reply:
Sure. Please notice this is a draft PR. I am still playing around with those settings.